From cac1f18c54c87839b4eaa812cd438dc4593c7975 Mon Sep 17 00:00:00 2001
From: tangwenji <tang.wenji@zte.com.cn>
Date: Thu, 15 Mar 2018 14:52:20 +0800
Subject: [PATCH] Fix default max_backup_files in ui_command_saveconfig

Signed-off-by: tangwenji <tang.wenji@zte.com.cn>
---
 targetcli/ui_root.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/targetcli/ui_root.py b/targetcli/ui_root.py
index a54845f..54d38e3 100644
--- a/targetcli/ui_root.py
+++ b/targetcli/ui_root.py
@@ -98,7 +98,7 @@ class UIRoot(UINode):
 
                     if backup_error == None:
                         # Kill excess backups
-                        max_backup_files = int(self.shell.prefs['max_backup_files'])
+                        max_backup_files = int(self.shell.default_prefs['max_backup_files'])
 
                         try:
                             with open(universal_prefs_file) as prefs:
-- 
1.8.3.1

