From a7254dfc15830eb64ae8d47639889cb6a23bd7f4 Mon Sep 17 00:00:00 2001
From: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>
Date: Tue, 12 Jun 2018 15:36:42 +0530
Subject: [PATCH] saveconfig: handle backups with block-level delete

Signed-off-by: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>
---
 targetcli/ui_backstore.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/targetcli/ui_backstore.py b/targetcli/ui_backstore.py
index 02695ba..7af2568 100644
--- a/targetcli/ui_backstore.py
+++ b/targetcli/ui_backstore.py
@@ -302,6 +302,10 @@ class UIBackstore(UINode):
             raise ExecutionError("No storage object named %s." % name)
 
         save = self.ui_eval_param(save, 'bool', False)
+        if save:
+            rn = self.get_root()
+            rn._save_backups(default_save_file)
+
         child.rtsnode.delete(save=save)
         self.remove_child(child)
         self.shell.log.info("Deleted storage object %s." % name)
-- 
1.8.3.1

